etl/helpers.py

```diff
@@ -263,3 +267,405 @@ def load_dependency(
             dataset = catalog.Dataset(dataset_path)
 
         return dataset
+
+
+# For backwards compatibility.
+Names = PathFinder
+
+
+def extract_step_attributes(step: str) -> Tuple[str, str, str, str, str, str, str]:
```

Comment:
This is gonna be useful for other parts of code. Would it be possible to return dictionary instead of 7-tuple? Working with tuples would become cumbersome if I needed only e.g. namespace.

Reply:
Thanks. Yes, I was thinking the same thing. I'll do that.

Reply:
Also would you mind moving it under `parse_step` [here](https://github.com/owid/etl/blob/master/etl/steps/__init__.py#L246)? It's more logical to have it together.

Reply:
Done.